### ISSUE TITLE
Error: RPC Validation Error

### ISSUE THREAD

**Issue by YYsuni** [MystenLabs/sui#12309]
![image](https://github.com/MystenLabs/sui/assets/53907086/c5e8b14d-b9e1-4432-ac03-d795f70b9e5c)

1. I use the [suins toolkit](https://github.com/SuiNSdapp/toolkit) to `getName` (e.g. 0x56b324de79947a817acec3e70c0981b567a985d7f9de809afe792088d7256228)

```ts
import { Connection, JsonRpcProvider } from '@mysten/sui.js'
import { SuinsClient } from '@/utils/suins-toolkit'

const provider = new JsonRpcProvider(new Connection({ fullnode: 'https://explorer-rpc.mainnet.sui.io/' }))
export const suinsClient = new SuinsClient(provider, {
	contractObjects: {
		registry: '0xe64cd9db9f829c6cc405d9790bd71567ae07259855f4fba6f02c84f52298c106',
		packageId: '0xd22b24490e0bae52676651b4f56660a5ff8022a2576e0089f79b3c88d44e08f0',
		suins: '',
		reverseRegistry: '0x2fd099e17a292d2bc541df474f9fafa595653848cbabb2d7a4656ec786a1969f'
	}
})

suinsClient.getName('0x56b324de79947a817acec3e70c0981b567a985d7f9de809afe792088d7256228')
```

2. Then it returns:
![image](https://github.com/MystenLabs/sui/assets/53907086/47b56152-fa99-4e4e-8016-7f393f27bbb0)

3. I find the warning logs occurring in `node_modules/@mysten/sui.js/src/rpc/client.ts`, the `ErrorResponse` does not match the return
```ts
export const ErrorResponse = object({
  jsonrpc: literal('2.0'),
  id: string(),
  error: object({
    code: any(),
    message: string(),
    data: optional(any()),
  }),
});
```

There are too many warnings because I requested a lot of address names on one page. Please fix it.

**Comment by Jordan-Mysten** [comment]
Fixed in #12347

**Comment by YYsuni** [comment]
Thanks for that. ❤️
